libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -496,19 +502,31 @@ internal class MapboxTripSession(
             if (!isActive) {
                 return@launch
             }
-            updateEnhancedLocation(status.enhancedLocation, status.keyPoints)
+            val enhancedLocation = status.navigationStatus.location.toLocation()
+            val keyPoints = status.navigationStatus.key_points.toLocations()
```

Comment:
Wondering if we should move these operations to `ioJobController` 🤔 

Same applies to other ones below 👇 

Any opinions @korshaknn @LukasPaczos?

Reply:
Those are rather light, I don't think it's worth it.